web/src/lib/i18n/en.json

```
@@ -902,6 +915,7 @@
   "validate": "Validate",
   "variables": "Variables",
   "version": "Version",
+  "version_announcement_message": "Hi friend, there is a new version of the application please take your time to visit the <link>release notes</link> and ensure your <code>docker-compose</code>, and <code>.env</code> setup is up-to-date to prevent any misconfigurations, especially if you use WatchTower or any mechanism that handles updating your application automatically.",
```

```suggestion
  "version_announcement_message": "Hi friend, there is a new version of the application please take your time to visit the <link>release notes</link> and ensure your <code>docker compose</code>, and <code>.env</code> setup is up-to-date to prevent any misconfigurations, especially if you use WatchTower or any mechanism that handles updating your application automatically.",
```
(unrelated to this PR)

---

I changed it to `docker-compose.yml` because `.env` is also a filename